### ISSUE TITLE
Problems with top margin gaps in web pages on iPhone X and later models

### ISSUE THREAD

**Issue by oink8407s** [gree/unity-webview#645]
![Image from iOS](https://user-images.githubusercontent.com/77921797/106862095-fe13a900-6709-11eb-8e8a-aee656d80704.png)

Hello, I'm Unity developer.
I'm using unity-webview well. Thank you, but there's a little problem using unity-webview. If you look at the top of the image, there is a gap when you create a web view from the iPhone X to the top model, and the image comes out after that. Can you tell me something about this problem?

**Comment by KojiNakamaru** [comment]
The image seems to be shown inside the webview, so the issue should be caused by html/javascript in which some adjustment for the header might be performed, depending on user-agent.

**Comment by oink8407s** [comment]
Yes, we thought it was a problem with publishing and tried to fix it. However, there was no problem with publishing and there was no problem with title margin in the actual iPhone safari, not webview in Unity. This is a problem that only occurs in the unity webview, and the same is happening in https://www.naver.com/,, https://www.daum.net/, and so on.
![KakaoTalk_20210205_152813858](https://user-images.githubusercontent.com/77921797/106997821-e5b59400-67c6-11eb-9d4f-7e48f329c598.jpg)

**Comment by KojiNakamaru** [comment]
Thank you for further examples! I could reproduce the issue with https://www.naver.com and fixed it in #646.  Could you please try the latest?

**Comment by oink8407s** [comment]
Yes, it's settled. It's coming out well. Thank you so much!
